### DISCUSSION TITLE
MQTT - Setting the active_state_timeout

### DISCUSSION THREAD

**Discussion by dsfsdbvxc** [mikebrady/shairport-sync#1863] [Q&A] [answered]
Firstly, appologies as this is probably a very simple mistake i'm making.

I've got Shairport-sync (4.3.3) working with MQTT this evening and its reporting its state correctly in Home Assistant. However i'm not sure how to change the active_state_timeout

In the [documentation](https://github.com/mikebrady/shairport-sync/blob/master/MQTT.md), under 'Overall Active States', where does this section go? In /etc/shairport-sync.conf ?

`sessioncontrol =
{
//	"active" state starts when play begins, and ends when the active_state_timeout has elapsed after play ends, unless another play session starts before the timeout has fully elapsed.
	active_state_timeout = 30.0;
};`

**Comment by dsfsdbvxc** [comment]
Sorted, see you can set in the Home Assistant configuration.yaml, under off_delay

**Reply by mikebrady** [reply, marked as answer]
Apologies for not responding — it’s good that you got it sorted.

But yeah, if you do a `# make install`, a sample configuration file is installed at `/etc/shairport-sync.conf.sample` and in it you can see where the `active_state_timeout` setting goes.

**Reply by dsfsdbvxc** [reply]
Thanks for the reply and extra info.

Also a big thank you for Shairport-sync. I've been using on 3 Pi's in my house for years and its been brilliant.
